tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -379,9 +384,9 @@ EOF
 (exit \$?)");
 
     # Setup access to Longhorn UI (useful for debugging)
-    assert_script_run("curl -kJLO https://gitlab.suse.de/virtualization/kubevirt-ci/-/raw/main/storage/longhorn-auth");
+    assert_script_run("curl -kJLO https://gitlab.suse.de/virtualization/kubevirt-ci/-/raw/c41969bca710335f7966b1588abca9958a33ff24/storage/longhorn-auth");
     assert_script_run("kubectl -n longhorn-system create secret generic basic-auth --from-file=auth=longhorn-auth || true");
-    assert_script_run("kubectl -n longhorn-system apply -f https://gitlab.suse.de/virtualization/kubevirt-ci/-/raw/main/storage/longhorn-ingress.yaml");
+    assert_script_run("kubectl -n longhorn-system apply -f https://gitlab.suse.de/virtualization/kubevirt-ci/-/raw/c41969bca710335f7966b1588abca9958a33ff24/storage/longhorn-ingress.yaml");
```

Comment:
and here, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19645/files#r1663631938

Reply:
It's the updated link from developer's suggestion.